src/util/ChartUtils.ts

```diff
@@ -213,6 +222,7 @@ export const getLegendProps = ({
         type: legendItem.props.iconType || legendType || 'square',
         color: getMainColorOfGraphicItem(item),
         value: name || dataKey,
+        // @ts-expect-error property strokeDasharray is required in Payload but optional in props
```

Comment:
😓 Do you think we could provide a reasonable default here if it is not passed? Could we even reuse an existing default?

Reply:
I wanted to avoid runtime changes.